src/Tags/Url.php

```diff
@@ -85,6 +88,13 @@ public function addImage(string $url, string $caption = '', string $geo_location
         return $this;
     }
 
+    public function addVideo(string $thumbnailLoc, string $title, string $description, $contentLoc = null, $playerLoc = null, array $options = [], array $allow = [], array $deny = []): static
```

Comment:
Don't abbreviate variables, use `thumbnailLocation`. Put each argument on it's own line,

Reply:
I kept the original name from the documentation: `<video:thumbnail_loc>` is the tag so I used `$thumbnailLoc` name. 

Changing names doesn't make people confused?

Reply:
And what do you exactly mean about "putting each argument on its own line"?